syntax = "proto3";
import "FlightState.proto";
import "AVStateFlags.proto";

package payload;

message AV_TO_GCS_DATA_2 {
    optional common.FlightState flightState = 1;
    optional common.AVStateFlags state_flags = 2;

    optional string GPS_latitude = 3;
    optional string GPS_longitude = 4;
}